runtests.py

```diff
@@ -42,7 +49,11 @@
             'polymorphic',
             'polymorphic.tests',
         ),
-        MIDDLEWARE_CLASSES=(),
+        MIDDLEWARE=(
+            'django.contrib.auth.middleware.AuthenticationMiddleware',
+            'django.contrib.messages.middleware.MessageMiddleware',
+            'django.contrib.sessions.middleware.SessionMiddleware',
+        ),
```

Comment:
hey, I have was wondering why this is defined now when `MIDDLEWARE_CLASSES` was empty before

Reply:
David tells me this is now required!